includes/components/class-admin.php

```diff
@@ -946,6 +949,261 @@ protected function get_extensions_tab() {
+			$post_types[] = [
+				'label' => sprintf( esc_html__( '%s:', 'hivepress' ), $post_type_count->type ),
+				'value' => $post_type_count->count,
+			];
+		}
+
+		// Get active plugins.
+		$active_plugins = array_filter(
+			array_map(
+				function ( $plugin ) {
+					$file = WP_PLUGIN_DIR . '/' . $plugin;
+
+					if ( file_exists( $file ) ) {
+						$headers = array_filter(
+							get_file_data(
+								$file,
+								[
+									'name'    => 'Plugin Name',
+									'version' => 'Version',
+									'url'     => 'Plugin URI',
+									'author'  => 'Author',
+								]
+							)
+						);
+
+						$value = [
+							'label' => sprintf( esc_html__( '%s:', 'hivepress' ), hp\get_array_value( $headers, 'name' ) ),
+							'value' => sprintf( esc_html__( 'by %1$s - %2$s', 'hivepress' ), hp\get_array_value( $headers, 'author' ), hp\get_array_value( $headers, 'version' ) ),
+							'url'   => hp\get_array_value( $headers, 'url' ),
+						];
+
+						return $value;
+					}
+				},
+				(array) get_option( 'active_plugins' )
+			)
+		);
+
+		// Get theme info.
+		$theme = wp_get_theme( get_template() );
+
+		$theme_info = [
+			[
+				'label' => esc_html__( 'Name:', 'hivepress' ),
+				'value' => $theme->get( 'Name' ),
+			],
+			[
+				'label' => esc_html__( 'Version:', 'hivepress' ),
+				'value' => $theme->get( 'Version' ),
+			],
+			[
+				'label' => esc_html__( 'Author URL:', 'hivepress' ),
+				'value' => $theme->get( 'AuthorURI' ),
+			],
+			[
+				'label' => esc_html__( 'Child theme:', 'hivepress' ),
+				'value' => is_child_theme() ? esc_html__( 'Yes', 'hivepress' ) : esc_html__( 'No', 'hivepress' ),
+			],
+		];
+
+		if ( is_child_theme() ) {
+			$parent_theme = wp_get_theme( $theme->template );
+
+			$theme_info = array_merge(
+				$theme_info,
+				[
+					[
+						'label' => esc_html__( 'Parent Theme Name', 'hivepress' ),
+						'value' => $parent_theme->name,
+					],
+					[
+						'label' => esc_html__( 'Parent Theme Version', 'hivepress' ),
+						'value' => $parent_theme->version,
+					],
+				]
+			);
+		}
+
+		$theme_info[] = [
+			'label' => esc_html__( 'HivePress support:', 'hivepress' ),
+			'value' => current_theme_supports( 'hivepress' ) ? esc_html__( 'Yes', 'hivepress' ) : esc_html__( 'No', 'hivepress' ),
+		];
+
+		// Get report.
+		$report = [
+			'content'  => '',
+			'sections' => [
+				[
+					'label'  => esc_html__( 'WordPress enviroment', 'hivepress' ),
+					'values' => [
+						[
+							'label' => esc_html__( 'HivePress version:', 'hivepress' ),
+							'value' => hivepress()->get_version(),
+						],
+						[
+							'label' => esc_html__( 'Action Scheduler package:', 'hivepress' ),
+							'value' => class_exists( 'ActionScheduler_Versions' ) && class_exists( 'ActionScheduler' ) ? \ActionScheduler_Versions::instance()->latest_version() . ' ' . \ActionScheduler::plugin_path( '' ) : esc_html__( 'No', 'hivepress' ),
```

Comment:
Please trim the plugin path, currently it shows full path from the server root which is not ok for security when shared on the forum